go/cmd/dolt/commands/sql.go

```diff
@@ -796,7 +813,11 @@ func runShell(ctx context.Context, se *engine.SqlEngine, mrEnv *env.MultiRepoEnv
 		//TODO: Handle comments and enforce the current line terminator
 		if matches := delimiterRegex.FindStringSubmatch(query); len(matches) == 3 {
```

Comment:
You might have to modify this regex based on the line terminator instead of directly calling TrimSuffix